package.json

```diff
@@ -18,23 +18,28 @@
   ],
   "scripts": {
     "build": "tsc --project tsconfig.json",
+    "build:watch": "pnpm build --watch",
     "dev": "nodemon dist/sprightly.js",
```

Comment:
never noticed this, why do we need it?

Reply:
Re-compiles the JS code on every change. Which in turn changes `dist/sprightly.js` that nodemon is watching for change when we used the `dev` script.